packages/foam-vscode/src/extension.ts

```diff
@@ -100,6 +100,13 @@ export async function activate(context: ExtensionContext) {
             'Foam: Reload the window to use the updated settings'
           );
         }
+      }),
+      // 2024-11-25 21:22:48 TODO wip not working as expected
+      workspace.createFileSystemWatcher('.gitignore').onDidChange(e => {
+        Logger.info(`[wtw] File changed: ${e.fsPath}`);
+        window.showInformationMessage(
+          'Foam: Reload the window to use the updated .gitignore settings'
+        );
```

Comment:
@riccardoferretti Hello, based on online searches and AI suggestions, I believe that this approach should be able to achieve the effect of issuing a prompt when modifying the ignore settings. However, when I tested it, I found that it did not take effect. I would like to consult with you to see if you have any experience and can help me identify where I might have gone wrong.

Reply:
Yeah that looks like it should work, I am not really sure why it doesn't...